api/paidAction/itemCreate.js

```diff
@@ -27,8 +27,8 @@ export async function getBaseCost ({ models, bio, parentId, subName }) {
       LEFT JOIN "Sub" s ON s.name = COALESCE(r."subName", i."subName")
       WHERE i.id = ${Number(parentId)}`
 
-    if (!sub) return DEFAULT_ITEM_COST
-    return satsToMsats(sub.replyCost)
+    if (sub?.replyCost) return satsToMsats(sub.replyCost)
```

Comment:
this will be false for 0 but `replyCost` can't be lower than 1 according to our formik validation